### DISCUSSION TITLE
Does zarr support include support for codecs?

### DISCUSSION THREAD

**Discussion by satra** [google/neuroglancer#571] [Q&A]
According to [this](https://github.com/google/neuroglancer/tree/master/src/datasource/zarr), zarr support does not extend to all of numcodecs and imagecodecs. Is that still the case? Are there any plans to support any of them?

cc: @balbasty

**Comment by jbms** [comment]
> According to [this](https://github.com/google/neuroglancer/tree/master/src/datasource/zarr), zarr support does not extend to all of numcodecs and imagecodecs. Is that still the case? Are there any plans to support any of them?
> 
> cc: @balbasty

We probably won't add all of them, but which codecs do you want in particular?

**Reply by satra** [reply]
jpegxl (in lossless and lossy form) is one we would probably want to use based on work that viren showed last year on denoising+jpegxl lossy compression.

**Reply by satra** [reply]
blosc is good currently for a bunch of use cases but the availability of a lossy codec like jpegxl would be helpful.
